runtime/src/main/java/org/corfudb/util/Utils.java

```diff
@@ -472,16 +472,19 @@ public static void printLogAnatomy(CorfuRuntime runtime, ILogData logData) {
     static TailsResponse getTails(Set<TailsResponse> responses) {
         long globalTail = Address.NON_ADDRESS;
         Map<UUID, Long> globalStreamTails = new HashMap<>();
+        long minEpoch = Layout.INVALID_EPOCH;
 
         for (TailsResponse res : responses) {
+            // All epochs should be equal as all the tails are queried using a single runtime layout.
+            minEpoch = Math.min(minEpoch, res.getEpoch());
```

Comment:
If all tails are expected to be computed with the same epoch, then why do we have a minEpoch ?